classes/controller/AdminController.php

```diff
@@ -2269,8 +2269,11 @@ public function initFooter()
         //rtl.js overrides inline styles
         //iso_code.css overrides default fonts for every language (optional)
         if ($this->context->language->is_rtl) {
-            $this->addJS(_PS_JS_DIR_.'rtl.js');
-            $this->addCSS(__PS_BASE_URI__.$this->admin_webpath.'/themes/'.$this->bo_theme.'/css/'.$this->context->language->iso_code.'.css', 'all', false);
+            $this->addJS(_PS_JS_DIR_.'rtl.js', true);
+            $css = __PS_BASE_URI__.$this->admin_webpath.'/themes/'.$this->bo_theme.'/css/'.$this->context->language->iso_code.'.css';
+            if (file_exists($css)) {
```

Comment:
We need to check it, if is exists because without this verification, we have an javascript error message in console. If you know another way, i'm listen :)